src/Conf.ml

```diff
@@ -1009,8 +1020,11 @@ let parse_line config ~verbose ~from s =
   | _ -> Error (`Malformed s)
 
 let is_project_root dir =
-  List.exists project_root_witness ~f:(fun name ->
-      Caml.Sys.file_exists (Filename.concat dir name) )
+  match !root with
+  | Some root -> Fpath.(equal (v dir |> normalize) (v root |> normalize))
```

Comment:
normalisation of the root should happen only once.
Also, how does that work when the root is not absolute ?

Reply:
Now it is normalized only once and it should be absolute (as well as the dir it is compared to).